### DISCUSSION TITLE
Hola si en el formulario ya se envió solo queda esperar?

### DISCUSSION THREAD

**Discussion by Lvelmo** [Launch-X-Latam/MisionFrontEnd#440] [Team Posts]
Hola entonces al concluir de llenar el formulario te envian el qr para entrar al Teams?

**Comment by sudoagile** [comment]
Yo encontré esto, ahí te explican todo en la primera clase, y creo que están las demás



De: Lvelmo ***@***.***>
Fecha: viernes, 7 de octubre de 2022, 23:28
Para:
CC: Subscribed ***@***.***>
Asunto: [Launch-X-Latam/mision-frontend] Hola si en el formulario ya se envió solo queda esperar? (#9)

Hola entonces al concluir de llenar el formulario te envian el qr para entrar al Teams?

**Comment by sudoagile** [comment]
https://web.microsoftstream.com/group/1cd6a6ad-7b5f-4845-b108-4f1ccb16d8dd?view=videos

De: Lvelmo ***@***.***>
Fecha: viernes, 7 de octubre de 2022, 23:28
Para:
CC: Subscribed ***@***.***>
Asunto: [Launch-X-Latam/mision-frontend] Hola si en el formulario ya se envió solo queda esperar? (#9)

Hola entonces al concluir de llenar el formulario te envian el qr para entrar al Teams?
